ngx-fudis/projects/ngx-fudis/src/lib/components/pagination/pagination.mdx

```diff
@@ -0,0 +1,46 @@
+import { Meta, Canvas, ArgTypes } from "@storybook/blocks";
+import { PaginationComponent } from "./pagination.component";
+import * as PaginationStories from "./pagination.stories";
+import { paginationExclude } from "../../utilities/storybook";
+
+<Meta title="Components/Pagination" />
+
+# Pagination
+
+Pagination is a navigation element that can be used on pages or elements. It provides controls for switching content across multiple pages.
+Use Pagination only for content which structure does not change between pages eg. tables and lists.
+
+## Example
+
+<Canvas of={PaginationStories.Example} />
+
+## Usage Guidelines
+
+```
+<fudis-pagination
+  [pageCount]="20"
+  [pageIndex]="0"
+  [pageHref]="(i) => '/products?page=' + (i + 1)"
+  [paginationAriaLabel]="'Example navigation'"
+  (pageChange)="onPageChange($event)">
+</fudis-pagination>
+
+```
+
+### Properties
+
+Pagination requires `pageCount` in order to generate total amount of pages. Additionally current `pageIndex` can be provided, the first page starts with index 0. Pagination has also a mandatory
```

Comment:
I would like to get a bit more specific documentation about `pageIndex`. I think it should clearly state what it does and as I understood it initializes the component with the given index and sets it as a starting page.

I would also organize the paragraph so that the mandatory properties come first and then the optional ones.